From f49b9e0e67746638e36b4b1ebf247643c82a8953 Mon Sep 17 00:00:00 2001
From: Gary Zhang <b13634@freescale.com>
Date: Fri, 17 Aug 2012 10:46:22 +0800
Subject: ENGR00220633 WM8962: add support for wm8962 in mx6sl evk

add support for audio codec wm8962 in mx6sololite evk board

Signed-off-by: Gary Zhang <b13634@freescale.com>
---
 sound/soc/imx/imx-wm8962.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/imx/imx-wm8962.c b/sound/soc/imx/imx-wm8962.c
index de0dcd6..8da589f 100644
--- a/sound/soc/imx/imx-wm8962.c
+++ b/sound/soc/imx/imx-wm8962.c
@@ -474,7 +474,7 @@ static int __init imx_asoc_init(void)
 
 	if (machine_is_mx6q_sabresd())
 		imx_dai[0].codec_name = "wm8962.0-001a";
-	else if (machine_is_mx6sl_arm2())
+	else if (machine_is_mx6sl_arm2() | machine_is_mx6sl_evk())
 		imx_dai[0].codec_name = "wm8962.1-001a";
 
 	imx_snd_device = platform_device_alloc("soc-audio", 5);
-- 
1.7.9.5

